homeassistant/components/onboarding/views.py

```diff
@@ -429,6 +436,65 @@ async def post(self, manager: BackupManager, request: web.Request) -> web.Respon
         return await self._post(request)
 
 
+class CloudForgotPasswordView(cloud_http.CloudForgotPasswordView):
+    """Login to Home Assistant Cloud."""
+
+    requires_auth = False
+    url = "/api/onboarding/cloud/forgot_password"
+    name = "api:onboarding:cloud:forgot_password"
+
+    def __init__(self, data: OnboardingStoreData) -> None:
+        """Initialize the view."""
+        self._data = data
+
+    async def post(self, request: web.Request) -> web.Response:
+        """Handle login request."""
+        if self._data["done"]:
```

Comment:
I'd add a comment why we do this. The `"done"` item holds the onboarding step names that are done, I think. Does it mean then that we want to check here if onboarding has started or not?